tensorflow_lite_support/ios/task/vision/sources/TFLObjectDetector.m

```diff
@@ -39,7 +40,7 @@ - (instancetype)init {
   return self;
 }
 
-- (nullable instancetype)initWithModelPath:(nonnull NSString *)modelPath {
+- (instancetype)initWithModelPath:(NSString *)modelPath {
```

Comment:
@khanhlvg I have removed the nullability of the return type here since the model path argument is non nullable by default and hence under no circumstances the created options can be null. But this holds true in swift since swift compiler generates an error if a null is passed. In objC only a warning is shown by the compiler. Theoretically you can still pass a null in ObjC and that ends up making the options non usable. So in such a case should we do a null check and return null for options or not handle this case?

Reply:
Also should we make options of all tasks subclasses of TFLBaseOptions? TFLBaseOptions is common for all tasks. In this way they'll all have a consistent initWithModelPath. So modifications can be done in one place.

Reply:
> Theoretically you can still pass a null in ObjC and that ends up making the options non usable. So in such a case should we do a null check and return null for options or not handle this case?

You can return a null object as if a null modelPath is passed to the initializer.

> Also should we make options of all tasks subclasses of TFLBaseOptions? TFLBaseOptions is common for all tasks. In this way they'll all have a consistent initWithModelPath. So modifications can be done in one place.

No. Let's keep the structure as it: ObjectDetectorOptions contains 2 properties: a TFLBaseOptions and a TFLClassificationOptions.. It aligns with the C++ and Python APIs.